<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>771</width>
    <height>690</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>771</width>
    <height>690</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>771</width>
    <height>690</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>calci</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.png</normaloff>icon.png</iconset>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>751</width>
      <height>671</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab1">
     <attribute name="title">
      <string>Quadratic Equations</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>411</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enter the co-efficients of quadratic equation &lt;span style=&quot; font-size:11pt; color:#22840c;&quot;&gt;axÂ²+bx+c=0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_fr">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>30</y>
        <width>101</width>
        <height>33</height>
       </rect>
      </property>
      <property name="text">
       <string>find roots</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_fb">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>161</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Feedback   :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>31</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>a =</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ln_a">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>30</y>
        <width>91</width>
        <height>33</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="ln_b">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>30</y>
        <width>91</width>
        <height>33</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="ln_c">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>30</y>
        <width>91</width>
        <height>33</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lb_x2">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>80</y>
        <width>31</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>, x =</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>40</y>
        <width>31</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>c =</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>40</y>
        <width>31</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>b =</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ln_info">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>110</y>
        <width>551</width>
        <height>33</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="ln_qe">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>70</y>
        <width>221</width>
        <height>33</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="lb">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Roots of the given Q.E </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>X-Y Plot      :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ln_root1">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>70</y>
        <width>101</width>
        <height>33</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>80</y>
        <width>61</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>are  x =</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_clr">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>30</y>
        <width>101</width>
        <height>33</height>
       </rect>
      </property>
      <property name="text">
       <string>clear</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ln_root2">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>70</y>
        <width>101</width>
        <height>33</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="GraphicsLayoutWidget" name="graphicsView">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>180</y>
        <width>711</width>
        <height>441</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab2">
     <attribute name="title">
      <string>Linear Equations</string>
     </attribute>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>491</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enter the co-efficients of linear equations  &lt;span style=&quot; font-size:11pt; color:#1a9511;&quot;&gt;a&lt;/span&gt;&lt;span style=&quot; font-size:11pt; color:#1a9511; vertical-align:sub;&quot;&gt;1&lt;/span&gt;&lt;span style=&quot; font-size:11pt; color:#1a9511;&quot;&gt;x+b&lt;/span&gt;&lt;span style=&quot; font-size:11pt; color:#1a9511; vertical-align:sub;&quot;&gt;1&lt;/span&gt;&lt;span style=&quot; font-size:11pt; color:#1a9511;&quot;&gt;y = c&lt;/span&gt;&lt;span style=&quot; font-size:11pt; color:#1a9511; vertical-align:sub;&quot;&gt;1&lt;/span&gt;  and  &lt;span style=&quot; font-size:11pt; color:#ee1166;&quot;&gt;a&lt;/span&gt;&lt;span style=&quot; font-size:11pt; color:#ee1166; vertical-align:sub;&quot;&gt;2&lt;/span&gt;&lt;span style=&quot; font-size:11pt; color:#ee1166;&quot;&gt;x+b&lt;/span&gt;&lt;span style=&quot; font-size:11pt; color:#ee1166; vertical-align:sub;&quot;&gt;2&lt;/span&gt;&lt;span style=&quot; font-size:11pt; color:#ee1166;&quot;&gt;y = c&lt;/span&gt;&lt;span style=&quot; font-size:11pt; color:#ee1166; vertical-align:sub;&quot;&gt;2&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>31</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>a1 =</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>50</y>
        <width>31</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>b1 =</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>50</y>
        <width>31</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>c1 =</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ln_2v_a1">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>40</y>
        <width>91</width>
        <height>33</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="ln_2v_b1">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>40</y>
        <width>91</width>
        <height>33</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="ln_2v_c1">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>40</y>
        <width>91</width>
        <height>33</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="ln_2v_a2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>80</y>
        <width>91</width>
        <height>33</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="ln_2v_b2">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>80</y>
        <width>91</width>
        <height>33</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="ln_2v_c2">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>80</y>
        <width>91</width>
        <height>33</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>31</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>a2 =</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>90</y>
        <width>31</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>b2 =</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>90</y>
        <width>31</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>c2 =</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>161</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Solution of Given L.E's </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ln_2v_eq1">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>120</y>
        <width>141</width>
        <height>33</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>130</y>
        <width>31</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>and</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ln_2v_eq2">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>120</y>
        <width>141</width>
        <height>33</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>130</y>
        <width>21</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>=&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_fs">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>40</y>
        <width>121</width>
        <height>33</height>
       </rect>
      </property>
      <property name="text">
       <string>Find Solution</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_clr2">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>80</y>
        <width>121</width>
        <height>33</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>170</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Feedback :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ln_info2">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>160</y>
        <width>621</width>
        <height>33</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="GraphicsLayoutWidget" name="graphicsView_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>210</y>
        <width>711</width>
        <height>411</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="ln_sol">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>120</y>
        <width>181</width>
        <height>33</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="btn_help">
    <property name="geometry">
     <rect>
      <x>468</x>
      <y>10</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Help</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_quit">
    <property name="geometry">
     <rect>
      <x>688</x>
      <y>10</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_about">
    <property name="geometry">
     <rect>
      <x>578</x>
      <y>10</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>About</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphicsLayoutWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
